### PR TITLE
CIT-41 #time 3h improve the logicical code of blog app in the myWeb p…

### PR DESCRIPTION
**Pull request by codal-uchoudhary**
…roject such as relation between modals and authentication

### CODE REVIEW

File: Projects/My_website/myWebsite/blog/views.py [hunk 1]
```diff
@@ -17,8 +18,9 @@ def posts(request):
 
 def postDetail(request,slug):
     iPost = Post.objects.all().get(slug = slug)
```
Comment (arvish-codal) [inline Projects/My_website/myWebsite/blog/views.py:20]: @codal-uchoudhary , are you sure this is the way we should be fetching an individual post instance?

**Review by arvish-codal — CHANGES_REQUESTED** [review]
@codal-uchoudhary  , are you sure this is the way we should be fetching an individual post instance as mentioned below ? 

>> Post.objects.all().get(slug=slug)

**Comment by codal-uchoudhary** [comment]
for now it is working perfectly. but i will find some efficient way to fetch out post in the today's PR.

**Comment by arvish-codal** [comment]
@codal-uchoudhary , why should be do all() if we want to fetch a unique Post ?? Why should we check each & every record.

can't we simply keep get(slug=slug) ?

**Review by arvish-codal — APPROVED** [review]

**Pull request merged**